lib/parsec/fileparse.py

```diff
@@ -289,7 +289,7 @@ def read_and_proc(fpath, template_vars=None, viewcfg=None, asedit=False):
             LOG.debug('Processing with Jinja2')
             try:
                 flines = jinja2process(flines, fdir, template_vars)
-            except (StandardError, TemplateError, UndefinedError) as exc:
+            except (Exception, TemplateError, UndefinedError) as exc:
```

Comment:
Does Exception not cover everything?

Reply:
I'm not sure what the original intention was here!

```suggestion
            except Exceptionas exc:
```

???

Reply:
Yes.